src/main/kotlin/org/jetbrains/research/ml/dataset/anonymizer/transformations/anonymization/java/JavaTypes.kt

```diff
@@ -0,0 +1,25 @@
+package org.jetbrains.research.ml.dataset.anonymizer.transformations.anonymization.java
+
+import com.intellij.psi.*
+import org.jetbrains.research.ml.dataset.anonymizer.transformations.anonymization.JvmTypes
+
+object JavaTypes : JvmTypes() {
+    override fun isClass(element: PsiElement?): Boolean = !isInterface(element) && element is PsiClass
```

Comment:
Maybe is it better to change the order: `element is PsiClass  && !isInterface(element)` ?